devel/ci/integration/tests/test_bodhi_cli.py

```diff
@@ -478,12 +478,13 @@ def test_updates_download(bodhi_container, db_container):
     # something we can track.
     koji_mock = "#!/bin/sh\necho TESTING CALL $0 $@\n"
     with replace_file(bodhi_container, "/usr/bin/koji", koji_mock):
-        result = _run_cli(bodhi_container, cmd)
+        with replace_file(bodhi_container, "/usr/local/bin/koji", koji_mock):
```

Comment:
do we use 2 different binary for the koji cli ?

Reply:
It seems that was f33/rawhide split? Or f32/f33? Can double-check.

Reply:
It's because of https://pagure.io/koji/issue/912
Koji it's installed from RPM package, but it isn't found because it doesn't provide egg info. So Bodhi will install Koji from Pypi and the Koji-Pypi executable is used.

Reply:
So, we keep the workaround? :-)

Reply:
Thanks @mattiaverga for finding that issue, I knew I saw it in the past :+1: 

Yeah let's keep the workaround, maybe we can add the koji issue as comment to explain why we did that ?

Reply:
Ok, will add Mattia's comment.